cloud-aws-cloudformation/src/test/java/com/sequenceiq/cloudbreak/cloud/aws/connector/resource/upgrade/operation/RdsInfoTest.java

```diff
@@ -0,0 +1,24 @@
+package com.sequenceiq.cloudbreak.cloud.aws.connector.resource.upgrade.operation;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import java.util.Map;
+
+import org.junit.jupiter.api.Test;
+
+public class RdsInfoTest {
+
+    @Test
+    void testCtorAndGetters() {
+        RdsState rdsState = RdsState.UNKNOWN;
+        Map<String, String> instancesToStatusMap = Map.of("instance", "status");
+        RdsEngineVersion rdsEngineVersion = new RdsEngineVersion("engineVersion");
+
+        RdsInfo rdsInfo = new RdsInfo(rdsState, instancesToStatusMap, rdsEngineVersion);
+
+        assertEquals(rdsInfo.getRdsState(), RdsState.UNKNOWN);
+        assertEquals(rdsInfo.getRdsEngineVersion(), rdsEngineVersion);
+        assertEquals(rdsInfo.getDbArnToInstanceStatuses(), instancesToStatusMap);
```

Comment:
nit: please swap these parameters as the first one should be the expected value.